deploy/helm/csi-s3/values.yaml

```
@@ -47,4 +47,6 @@ tolerations:
 
 nodeSelector: {}
 
+priorityClassName: ""
```

IMO This should by system-node-critical by default
```suggestion
priorityClassName: "system-node-critical"
```